apps/rush-lib/assets/rush-init/common/config/rush/command-line.json

```diff
@@ -167,20 +167,52 @@
        * (Required) A list of custom commands and/or built-in Rush commands that this parameter may
        * be used with.  The parameter will be appended to the shell command that Rush invokes.
        */
-       "associatedCommands": [ "build", "rebuild" ]
+      "associatedCommands": [ "build", "rebuild" ]
     },
 
     {
       /**
        * (Required) Determines the type of custom parameter.
-       * A "flag" is a custom command-line parameter whose presence acts as an on/off switch.
+       * A "string" is a custom command-line parameter whose value is a simple text string.
+       */
+      "parameterKind": "string",
+      "longName": "--my-string",
+      "description": "A custom string parameter for the \"my-global-command\" custom command",
+
+      "associatedCommands": [ "my-global-command" ],
```

Comment:
Documentation?

Reply:
It is documented [above](https://github.com/microsoft/rushstack/blob/b971a28b400baaf992f95db195c832341aa29245/apps/rush-lib/assets/rush-init/common/config/rush/command-line.json#L167).

To avoid copy+pasting large comments, in this file we only document the first occurrence of a JSON field.